docs/api/declarations.rst

```diff
@@ -37,15 +37,15 @@ implementer_only
 implements
 ----------
 
-(The `implementer` decorator is preferred to this.)
+(Does not work on Python 3.  Use the `implementer` decorator instead.)
```

Comment:
> (These notes should probably use some Sphinx directive to stand out
better, but I'm not well versed in Sphinx and I don't know which one I
should use.)

`.. caution:: Does not work on Python 3.  Use the `implementer` decorator instead.` is one way. 

(Themes usually style `.. note::` and `.. caution::`. They may or may not style `.. warning::`. They usually don't style `.. important::` and `.. tip::` in my experience.)